department-of-veterans-affairs/va.gov-cms
# VAMC-11201: VISN 1, VISN 5 and VISN 6 systems menus.

## Description

Map out and Create Admin and Editorial Menus for VISN 1, 5 and 6 Systems.

## QA steps

As admin

- [x] Visit <Manage/content/menus/VAMC system menus> and validate that menus displayed match this
<img width="1666" alt="Screen Shot 2020-07-30 at 11 38 26 AM" src="https://user-images.githubusercontent.com/56320995/88945537-1aff8900-d25c-11ea-8e5c-9c782acb9c05.png">


Visit the following
- [x] /admin/structure/types/manage/health_care_local_facility
- [x] /admin/structure/types/manage/health_care_local_health_service
- [x]  /admin/structure/types/manage/health_care_region_page
- [x] /admin/structure/types/manage/regional_health_care_service_des
- [x] /admin/structure/types/manage/vamc_operating_status_and_alerts
and validate that the content types have new Menu items (VISN 1, 5 and 6 systems) selected as enabled (check box) to match this.
<img width="1636" alt="Screen Shot 2020-07-30 at 11 40 55 AM" src="https://user-images.githubusercontent.com/56320995/88945690-4b472780-d25c-11ea-9e15-ab2cac49d9ff.png">

- [x] Visit "Sections/veterans health administration/VAMC facilities" and validate that menus displayed for VISN 1, 5 and 6 match this.
<img width="1676" alt="Screen Shot 2020-07-21 at 2 54 39 PM" src="https://user-images.githubusercontent.com/56320995/88094954-5a461f80-cb62-11ea-9155-5fad9a5f2698.png">
<img width="1661" alt="Screen Shot 2020-07-30 at 11 42 59 AM" src="https://user-images.githubusercontent.com/56320995/88945811-75004e80-d25c-11ea-8688-8db377232bde.png">
<img width="1636" alt="Screen Shot 2020-07-30 at 11 44 08 AM" src="https://user-images.githubusercontent.com/56320995/88945868-82b5d400-d25c-11ea-9369-d848c2a89cf8.png">

- [x] Visit "Manage/Content/Menus/CMS administration/Administration" and validate that VISN 1, 5 and 6 systems are assigned as child.
<img width="1680" alt="Screen Shot 2020-07-21 at 3 00 54 PM" src="https://user-images.githubusercontent.com/56320995/88095432-1e5f8a00-cb63-11ea-9c51-c9d38586f338.png">
<img width="1676" alt="Screen Shot 2020-07-30 at 11 46 43 AM" src="https://user-images.githubusercontent.com/56320995/88945987-a711b080-d25c-11ea-97e9-c91e94ef2c44.png">
<img width="1658" alt="Screen Shot 2020-07-30 at 11 47 53 AM" src="https://user-images.githubusercontent.com/56320995/88946006-aed15500-d25c-11ea-9f3a-7eb2aa4d6105.png">

- [x] Visit "/admin/config/user-interface/menu-breadcrumb" and validate VISN 1, 5 and 6 systems are enabled to match this.
<img width="1661" alt="Screen Shot 2020-07-30 at 11 51 21 AM" src="https://user-images.githubusercontent.com/56320995/88945087-8ac14400-d25b-11ea-9a2d-a0846e3364e8.png">


## Definition of Done
